File: BEE 1048.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        double num = input.nextDouble();
        double earned;
        if (num >= 0 && num <= 400){
            earned = num * (15.0/100);
            System.out.printf("Novo salario: %.2f\n", (earned + num));
            System.out.printf("Reajuste ganho: %.2f\n", earned);
            System.out.println("Em percentual: 15 %");
        }
        else if (num > 400 && num <= 800){
            earned = num * (12.0/100);
            System.out.printf("Novo salario: %.2f\n", (earned + num));
            System.out.printf("Reajuste ganho: %.2f\n", earned);
            System.out.println("Em percentual: 12 %");
        }
        else if (num > 800 && num <= 1200){
            earned = num * (10.0/100);
            System.out.printf("Novo salario: %.2f\n", (earned + num));
            System.out.printf("Reajuste ganho: %.2f\n", earned);
            System.out.println("Em percentual: 10 %");
        }
        else if (num > 1200 && num <= 2000){
            earned = num * (7.0/100);
            System.out.printf("Novo salario: %.2f\n", (earned + num));
            System.out.printf("Reajuste ganho: %.2f\n", earned);
            System.out.println("Em percentual: 7 %");
        }
        else{
            earned = num * (4.0/100);
            System.out.printf("Novo salario: %.2f\n", (earned + num));
            System.out.printf("Reajuste ganho: %.2f\n", earned);
            System.out.println("Em percentual: 4 %");
        }
    }
}

File: BEE 1048.py
num = float(input())

earned = 0.0

if (num >= 0 and num <= 400):
    earned = num * (15.0/100)
    print("Novo salario: {:.2f}".format((earned + num)))
    print("Reajuste ganho: {:.2f}".format(earned))
    print("Em percentual: 15 %")

elif (num > 400 and num <= 800):
    earned = num * (12.0/100)
    print("Novo salario: {:.2f}".format((earned + num)))
    print("Reajuste ganho: {:.2f}".format(earned))
    print("Em percentual: 12 %")

elif (num > 800 and num <= 1200):
    earned = num * (10.0/100)
    print("Novo salario: {:.2f}".format((earned + num)))
    print("Reajuste ganho: {:.2f}".format(earned))
    print("Em percentual: 10 %")

elif (num > 1200 and num <= 2000):
    earned = num * (7.0/100)
    print("Novo salario: {:.2f}".format((earned + num)))
    print("Reajuste ganho: {:.2f}".format(earned))
    print("Em percentual: 7 %")
    
else:
    earned = num * (4.0/100)
    print("Novo salario: {:.2f}".format((earned + num)))
    print("Reajuste ganho: {:.2f}".format(earned))
    print("Em percentual: 4 %")